src/Java/gtPlusPlus/xmod/gregtech/HANDLER_GT.java

```diff
@@ -145,7 +145,7 @@ public static void postInit(){
 
 	public static void onLoadComplete(FMLLoadCompleteEvent event) {
 		removeCrudeTurbineRotors();
-		cleanAssemblyLineRecipeMap();		
+		//cleanAssemblyLineRecipeMap();
```

Comment:
what is that?

Reply:
it mess up the assline recipe display in nei, making the circuits can't cycle in nei